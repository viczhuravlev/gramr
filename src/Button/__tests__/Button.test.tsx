import React from 'react';

import { shallow } from 'enzyme';
import { Button } from '../index';

describe('My Button', () => {
  it('renders button', () => {
    const wrapper = shallow(<Button text={'Text for button'} />);

    console.log(wrapper);
    expect(wrapper).toMatchSnapshot();

    // expect(wrapper.find('h1').html()).toMatch(/Hello, Enzyme/)
  });

  it('renders', () => {
    const wrapper = shallow(
      <div>
        <h1>Hello, Enzyme!</h1>
      </div>
    );
    expect(wrapper.find('h1').html()).toMatch(/Hello, Enzyme/);
  });

  it('renders snapshots, too', () => {
    const wrapper = shallow(
      <div>
        <h1>Hello, Enzyme!</h1>
      </div>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
