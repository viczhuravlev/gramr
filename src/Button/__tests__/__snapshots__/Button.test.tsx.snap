// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Button renders button 1`] = `
<button>
  Text for button
</button>
`;

exports[`My Button renders snapshots, too 1`] = `
<div>
  <h1>
    Hello, Enzyme!
  </h1>
</div>
`;
